test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
The change I think this PR introduces is that `package` namespaces are actually honoured unlike before where they were entirely ignored. This would mean that it could cause confusion to someone use `package` in their `.proto` files and hoping they were ignored. I think the reason this is a little confusing is that the examples in the tests currently include `package MumbleProto` which is just being ignored but it's not obvious because `MumbleProto` just happens to also be the name of the module in which we have `use Protobuf`.